fe1-web/src/features/rollCall/screens/RollCallOpened.tsx

```diff
@@ -104,9 +104,8 @@ const RollCallOpened = () => {
   };
 
   const onCloseRollCall = () => {
-    const updateId = Hash.fromStringArray(EventTags.ROLL_CALL, lao.id.toString(), rollCallID, time);
     const attendeesList = Array.from(attendees).map((key: string) => new PublicKey(key));
-    return requestCloseRollCall(updateId, attendeesList)
```

Comment:
I think it was a great idea to remove the `updateId` from here, but I would suggest moving it into `requestCloseRollCall`, by passing it the `lao.id`and `rollCallID`, instead of just an externally-calculated updateId.

This also ensures that whoever calls `requestCloseRollCall` will always compute the hash in the same way (correctly, hopefully 😅 )